spec/lib/twingly/url_spec.rb

```diff
@@ -564,12 +565,6 @@ def leading_and_trailing_whitespace
       it { is_expected.to eq(expected) }
     end
 
-    context "oddly enough, does not alter URLs with consecutive dots" do
```

Comment:
hehe, the quirk with `twingly-url` is that is was intentionally meant to mimic the behavior of the good ol' .NET implementation, the change here seems to make the two implementation diverge as I gather, not sure it matters anymore though 🤷

Reply:
Just mentioning that normalization of links is/used to be a thing, in those cases it didn't necessarily mean that they'd have to be "visitable".